### PR TITLE
[script.service.hue] 1.1.1

### PR DESCRIPTION
**Pull request by zim514**
### Description

**v1.1**

- Add toggle to disable Hue Labs when Ambilight is activated (Thanks @toupeira!)
- Refactoring
- Bug and crash fixes
- Update Qhue lib
- Improve exception handling
- Add localisations
- Refactor logging and settings
- Update to new settings system

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by zim514** [comment]
Update of german .po files, no code changes since last push.

File: script.service.hue/resources/lib/hue.py [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+from .settings import validate_settings
+
+
+def create_hue_scene(bridge):
+    xbmc.log("[script.service.hue] In kodiHue createHueScene")
+    scenes = bridge.scenes
+
+    xbmcgui.Dialog().ok(heading=_("Create New Scene"), message=_("Adjust lights to desired state in the Hue App to save as new scene.[CR]Set a fade time in seconds, or 0 for an instant transition."))
+
+    scene_name = xbmcgui.Dialog().input(_("Scene Name"))
```
[the remark this answers is not in the document]
Comment (zim514) [reply script.service.hue/resources/lib/hue.py:23]: Please see https://github.com/xbmc/repo-scripts/pull/2105#discussion_r730342718

File: script.service.hue/resources/lib/hue.py [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+
+        if selected:
+            result = scenes(lights=selected, name=scene_name, recycle=False, type='LightScene', http_method='post', transitiontime=transition_time)
+            # xbmc.log("[script.service.hue] In kodiHue createHueScene. Res: {}".format(res))
+            if result[0]["success"]:
+                xbmcgui.Dialog().ok(heading=_("Create New Scene"), message=_("Scene successfully created![CR]You may now assign your scene to player actions."))
+            else:
+                xbmcgui.Dialog().ok(_("Error"), _("Scene not created."))
+    else:
+        xbmcgui.Dialog().ok(_("Error"), _("Scene not created."))
```
[the remark this answers is not in the document]
Comment (zim514) [reply script.service.hue/resources/lib/hue.py:43]: See https://github.com/xbmc/repo-scripts/pull/2105#discussion_r730342718

**Comment by ronie** [comment]
@enen92 good to go?

**Pull request closed without merging**